schema.sql

```diff
@@ -69,7 +69,11 @@ CREATE TABLE genotypes (
        quality TEXT,
 
        -- Cycledash-derived data
-       "annotations:gene_names" TEXT,
+       -- These are from the varcode annotation worker
+       "annotations:gene_name" TEXT,
```

Comment:
I don't have a very string opinion about this, but could you file this in an issue to resolve out of band with this PR?